server/api/api/endpoints/events.py

```diff
@@ -61,7 +61,7 @@ async def post_event(
             project=project, name=name, request=request, json=data
         )
 
-    logger.debug("Got event", project=project, name=name)
+    logger.debug("Got event", project=project, name=name, id=event_data.entity.ids[0])
```

Comment:
might be worth to log all ids for the future in case we'll have multiple entities

Reply:
the event will be on a specific id, so it will be wrong to log all the ids here.. in the future when we will have multiple entities we will have to add a support for that case and log only the relevant id.